ocr.py

```diff
@@ -1,13 +1,25 @@
 #!/usr/bin/env python
 import requests
 import time
+import io
+from PIL import Image
+
+def scaleImage(imgFile, maxSize):
```

Comment:
Can we have `imgPath` instead of `imgFile` as a parameter? `imgFile` is a little misleading; I thought it was expecting a `file` object from `request.files['my-file']`.